components/common/GlobalNav.tsx

```diff
@@ -563,18 +593,26 @@ function GlobalNav(props) {
                   )
                 )}
               {plan.features.enableSearch && (
-                <NavItem className="d-md-none">
+                <NavItem className="d-md-none mb-2">
                   <NavLink>
                     <NavigationLink slug="/search" onClick={handleClose}>
-                      <NavHighlighter>
-                        <Icon name="search" className="me-2" />
+                      <NavHighlighter className="highlighter">
+                        <Icon
+                          name="search"
+                          className="me-2"
+                          width="1.75rem"
+                          height="1.75rem"
+                        />
                         {t('search')}
                       </NavHighlighter>
                     </NavigationLink>
                   </NavLink>
                 </NavItem>
               )}
               <LanguageSelector mobile="true" />
+              {customToolbarItems.length > 0 && (
+                <CustomToolbar items={customToolbarItems} mobile="true" />
```

Comment:
Should `mobile` be a boolean rather than a string?

```suggestion
                <CustomToolbar items={customToolbarItems} mobile />
```

Reply:
Yes!